examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
ok cool - do you think this is done anywhere else in the code / examples? I
can go hunting if need be, but it might be worth noting to the user somehow
(either via logging or via example) that if they do things to individual
channels, and then apply projections, then it'll at the least mix in other
stuff...

And just to confirm, the right thing to do here is do a notch filter on all
channels, and then under the hood call inst.copy().apply_proj(), right?

On Thu, Dec 31, 2015 at 12:03 PM, Eric Larson <notifications@github.com>
wrote:

> In examples/time_frequency/plot_compute_raw_data_spectrum.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48665421>:
>
> > @@ -70,3 +71,18 @@
> >
> >  ax.set_title('Four left-temporal magnetometers')
> >  plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
> > +
> > +# Alternatively, you may also create PSDs from Raw objects with psd_XXX
> > +f, ax = plt.subplots()
> > +psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
>
> Yes, a projection operator linearly mixes channels. Good catch.
>
> If we want both notch and proj, we should notch *all* channels used in
> projection. The way we were doing it before was wrong -- notching 4
> channels then applying proj *should* make the effect of the notch almost
> disappear. So your new code snippet above is the right thing to do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48665421>.
>